bin/wasm-node/rust/src/json_rpc_service.rs

```diff
@@ -282,7 +315,12 @@ impl JsonRpcService {
     ///
     /// Depending on the request, either calls [`JsonRpcService::send_back`] immediately or
     /// spawns a background task for further processing.
-    async fn handle_rpc(self: Arc<JsonRpcService>, request_id: &str, call: MethodCall) {
+    async fn handle_rpc(
+        self: Arc<JsonRpcService>,
+        request_id: &str,
+        call: MethodCall,
+        source_id: u32,
```

Comment:
```suggestion
        source_id: u32,
        request_id: &str,
        call: MethodCall,
```

This is a bit nit-picky, but putting arguments in order of importance isn't a bad idea.